### PR TITLE
Improve docs for Running custom python scripts

### PR DESCRIPTION
**Pull request by lalongooo**

### CODE REVIEW

File: scrapy-cloud.rst [hunk 1]
```diff
@@ -235,7 +235,7 @@ You can also setup periodic jobs to run the script or do it via the API.
 
 For running the script through the API, you need to use a private API (which may subject to change in the future). Here is an example using ``curl``::
 
-    curl -X POST -d '{"job_cmd": ["py:hello.py"]}' https://storage.scrapinghub.com/jobq/1/push
+    curl-u API_KEY: -X POST -d '{"job_cmd": ["py:hello.py"]}' https://storage.scrapinghub.com/jobq/PROJECT_ID/push
```
Comment (pablohoffman) [inline scrapy-cloud.rst]: missing space between curl and -u

Comment (lalongooo) [inline scrapy-cloud.rst]: Fixed @pablohoffman ... 😄

**Comment by lalongooo** [comment]
Hey @pablohoffman, is everything ok with this PR?

**Comment by pablohoffman** [comment]
yeah, thanks @lalongooo !

**Pull request merged**